tests/rados/pool_tests.py

```diff
@@ -316,11 +319,26 @@ def run(ceph_cluster, **kw):
             log.debug(
                 f"PG count on pool {pool_name} post addition of bulk flag : {pg_count_bulk_true}"
             )
-            if pg_count_bulk_true < init_pg_count:
-                raise Exception(
-                    f"Actual pg_num post bulk enablement {pg_count_bulk_true}"
-                    f" is expected to be greater than {init_pg_count}"
+            no_count_change = False
+            if pg_count_bulk_true <= init_pg_count:
+                log.info(
+                    "Checking the ideal PG count on the pool and the threshold set. default is 3"
                 )
+                pool_data = rados_obj.get_pg_autoscale_status(pool_name=pool_name)
+                final_pg_num = pool_data["pg_num_final"]
+                log.debug(
+                    f"Pool : {pool_name} details as fetched from autoscale-status : {pool_data}"
+                )
+                if float(final_pg_num / init_pg_count) <= 3.0:
```

Comment:
Please let me know what is the value 3.0.

Reply:
It's the threshold value, that is set for the determination of wether the PGs on a pool should be increased or not. 

I added the doc in the comments for reference.